core/server/data/migrations/versions/2.0/2-update-posts.js

```diff
@@ -36,6 +42,7 @@ module.exports.up = (options) => {
         });
 };
 
+// @NOTE: all posts are getting updated in a transaction. MySQL will auto-rollback the changes from above.
 module.exports.down = () => {
```

Comment:
We should implement the down hook anyway, because if people migrate to 2.0 and they detect a critical problem with the posts, they should be able to rollback to 1.x. This requires to keep all the version logic (v1 and v2) i guess. Haven't thought about it in detail.

Reply:
You migrated from Ghost 1.0 to Ghost 2.0. Is it possible to differentiate:

- posts which were originally old editor posts (rollback to old html format)
- posts which were originally Koenig Beta posts (do nothing)

cc @kevinansfield